bin/crew

```diff
@@ -1279,38 +1289,17 @@ def install_package(pkgdir)
         end
       end
     end
-    if File.file?("#{CREW_PREFIX}/bin/rdfind")
+
+    if File.executable?("#{CREW_PREFIX}/bin/rdfind")
       puts 'Using rdfind to convert duplicate files to hard links.'
       system 'rdfind -removeidentinode true -makesymlinks false -makehardlinks true -makeresultsfile false .'
     end
 
-    # check if the available rsync command support ACLs
+    install_files(File.join('.', HOME)) if Dir.exist?(File.join('.', HOME))
+    install_files(File.join('.', CREW_PREFIX)) if Dir.exist?(File.join('.', CREW_PREFIX))
 
-    rsync_available = true if File.file?("#{CREW_PREFIX}/bin/rsync") && `#{CREW_PREFIX}/bin/rsync --version`.include?('rsync  version')
-    puts 'rsync is not working. Please (re)install it with \'crew remove musl_zstd zstd ; crew install musl_zstd ; crew install rsync\''.lightred unless rsync_available || (@pkg.name == 'rsync')
-    if Dir.exist? "#{pkgdir}/#{HOME}"
-      if rsync_available
-        system "rsync -ahHAXW --remove-source-files ./#{HOME.delete_prefix('/')}/ #{HOME}"
-      else
-        system "tar -c#{@verbose}f - ./usr/* | (cd /; tar xp --keep-directory-symlink -f -)"
-      end
-    end
-    if Dir.exist? "#{pkgdir}/usr/local"
-      if rsync_available
-        # Adjust "./usr/local" if the build CREW_PREFIX ever changes.
-        system "rsync -ahHAXWx --remove-source-files ./usr/local/ #{CREW_PREFIX}"
-      else
-        system "tar -c#{@verbose}f - ./usr/* | (cd /; tar xp --keep-directory-symlink -f -)"
-      end
-    end
-    if Dir.exist? "#{pkgdir}/#{CREW_PREFIX}"
-      if rsync_available
-        # Adjust "./usr/local" if the build CREW_PREFIX ever changes.
-        system "rsync -ahHAXWx --remove-source-files ./#{CREW_PREFIX}/ #{CREW_PREFIX}"
-      else
-        system "cp -a#{@verbose} ./#{CREW_PREFIX}/* #{CREW_PREFIX}"
-      end
-    end
+    # Adjust "./usr/local" if the build CREW_PREFIX ever changes.
+    install_files('./usr/local', CREW_PREFIX) if Dir.exist?('./usr/local') && CREW_PREFIX != '/usr/local'
```

Comment:
What is wrong with simply running `install_files(CREW_PREFIX)`?

Reply:
This line is used to move `/usr/local` to custom `CREW_PREFIX` in case `CREW_PREFIX != /usr/local` (inherited from the original logic)

Reply:
Why don't we just install directly in CREW_PREFIX instead of moving things around?

Reply:
As this part also handles copying prebuilt binaries after moving out `filelist/directorylist` (which assume CREW_PREFIX is `/usr/local` when packaging)

Reply:
Why make that assumption at all?  Just extract directly to CREW_PREFIX.

Reply:
Oh, so you are saying the `usr/local` path is in the archive?  Why don't we build binaries relative to `CREW_PREFIX` then?

Reply:
Let's make changes in the another pr?

Reply:
Let's make that PR, though, as this breaks installing on non `/usr/local` `CREW_PREFIX`'s.

Both on my refactor and the original.

Reply:
This line is no longer necessary if we adjust the `dst` parameter as commented above?

Reply:
I'm thinking lines 1318 and 1321 can be one call as below?
```
install_files('./usr/local')
```
This is assuming the `dst` parameter is changed as suggested above in the `install_files` function.